src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi

```diff
@@ -145,19 +144,10 @@ cdef class _AioCall:
                receive_initial_metadata_op, receive_message_op,
                receive_status_on_client_op)
 
-        try:
-            await execute_batch(self._grpc_call_wrapper,
-                                        ops,
-                                        self._loop)
-        except asyncio.CancelledError:
-            status = self._cancel_and_create_status(cancellation_future)
-            initial_metadata_observer(None)
-            status_observer(status)
-            raise
-        else:
-            initial_metadata_observer(
-                receive_initial_metadata_op.initial_metadata()
-            )
+        # Executes all operations in one batch.
+        await execute_batch(self._grpc_call_wrapper,
+                            ops,
+                            self._loop)
```

Comment:
Im still thinking that we would need to do our best here and try to cancel the call if it was not cancelled explicitly by the user, so having something like this

```python
try:
     await execute_batch(...)
except asyncio.CancelledError:
    if not self._is_locally_cancelled:
        grpc_call_cancel(...)
    raise
````